practicals/2-od.Rmd

```diff
@@ -1,127 +1,178 @@
 subtitle: '<br/>Practical'
-author: "Robin Lovelace"
+author: "Malcolm Morgan"
 date: 'University of Leeds `r # Sys.Date()()`<br/><img class="img-footer" alt="" src="https://comms.leeds.ac.uk/wp-content/themes/toolkit-wordpress-theme/img/logo.png">'
 output:
   github_document:
     number_sections: true
 bibliography: ../tds.bib
 ---
 
+# Review Homework
+
+You should now be familiar with the basics of R and the `tidyverse`. If you have not completed these tasks go back and do them first:
+
+- Read Chapters 2, 3, and 4 of [Reproducible road safety research with R](https://itsleeds.github.io/rrsrr/basics.html)
+- Read Chapters 3 and 5 of [R for Data Science](https://r4ds.had.co.nz/data-visualisation.html)
+
+# Getting started with GIS in R
+
+Note that this practical takes sections from Chapters 2 - 8 of [Geocomputation with R](https://r.geocompx.org). You should expand your knowledge by reading these chapters in full.
 
 
 ## Pre-requisites {-}
 
 You need to have a number of packages installed and loaded.
-Install the packages by typing in the following commands into RStudio (you do not need to add the comments after the `#` symbol):^[
-Note: if you want to install the development version of a package from GitHub, you can do so.
-Try, for example, running the following command: `remotes::install_github("ITSLeeds/pct")`
-]
+Install the packages by typing in the following commands into RStudio (you do not need to add the comments after the `#` symbol)
 
-```{r, eval=FALSE}
-install.packages("remotes")
-pkgs = c(
-  "nycflights13",# data package
-  "pct",         # package for getting travel data in the UK
-  "sf",          # spatial data package
-  "stats19",     # downloads and formats open stats19 crash data
-  "stplanr",     # for working with origin-destination and route data
-  "tidyverse",   # a package for user friendly data science
-  "tmap"         # for making maps
-)
-remotes::install_cran(pkgs)
-remotes::install_github("nowosad/spDataLarge")
-```
+If you need to install any of these packages use:
 
-Load the tidyverse package as follows:
+```{r echo = T, results = 'hide', eval = FALSE}
+install.packages("sf")  # Install a package from CRAN
+remotes::install_github("Nowosad/spDataLarge")   # install from GitHub using the remotes package
+```
 
-```{r}
-library(tidyverse)
+```{r echo = T, results = 'hide', warning=FALSE, message=FALSE}
+library(sf)          # vector data package 
+library(tidyverse)   # tidyverse packages
 ```
 
-# Project set-up and tidyverse testing
+- It relies on **spData**, which loads datasets used in the code examples of this chapter:
+
+```{r 03-attribute-operations-2, echo = T, results = 'hide',warning=FALSE, message=FALSE}
+library(spData)  # spatial data package 
+```
 
-1. Check your packages are up1.to-date with `update.packages()`
+1. Check your packages are up-to-date with `update.packages()`
 1. Create an RStudio project with an appropriate name for this module (e.g. `TDSmodule`)
```

Comment:
```suggestion
1. Create an RStudio project with an appropriate name for this session (e.g. `practical2`)
```

The old one doesn't really follow from last week when they created a project called `practical`:

https://github.com/ITSLeeds/TDS/blob/28205a0d1d3766a56afae96c876448ef2f5e4601/practicals/1-intro.Rmd#L120